backend/src/crd/controller/viewer/main.go

```diff
@@ -76,8 +76,7 @@ func main() {
 		log.Fatal(err)
 	}
 
-	_, err = builder.SimpleController().
-		WithManager(mgr).
+	_, err = builder.ControllerManagedBy(mgr).
```

Comment:
SimpleController is deprecated  https://github.com/kubernetes-sigs/controller-runtime/pull/542